### ISSUE TITLE
Zadanie domowe - zdjęcia

### ISSUE THREAD

**Issue by Mathew582** [daftcode/frontend4beginners2018#3]
Cześć, mam pytanie dotyczące wstawiania zdjęć. W kodzie wydaje mi się wszystko w porządku jednak przy podglądzie strony w przeglądarce zdjęcia się nie wczytują wgl :( ktoś może wie dlaczego?
To fragment Kodu:

`<!DOCTYPE html>
<html>
    <head>
        <title>Strona zespołu Zeplin</title>
    </head>
    <body>
        <img class="image" scr="obrazki/logo.png" alt="logo zespołu" />
`

A to umiejscowienie pliku txt oraz zdjęcia logo.png, które nie chce się wczytać (jest w folderze obrazki)
![image](https://user-images.githubusercontent.com/44846138/48305531-8fe0db80-e52c-11e8-82c6-377a1a3a2135.png)

**Comment by nekorushi** [comment]
Literówki to raz, natomiast dodatkowo plik z kodem strony powinien być zapisany z rozszerzeniem "html" a nie "txt".

**Comment by carterstorm** [comment]
Wydaje mi się, że zdjęcia trzeba powklejać. Co do kolorów fontów i samego background to już raczej w CSS :>

**Comment by nekorushi** [comment]
Tak, zdjęcia należy wstawić w znacznikach `<img>`. CSS będziemy pisać w następnych etapach :)

**Comment by antonNina** [comment]
Hej! Mam kilka pytań, jeśli chodzi o ostateczną formę pierwszego zadania domowego. 
To ma być tylko (brzydka i biała ;-) ) struktura/szkielet html całej strony, czy coś więcej? 
Z tego, co zrozumiałam kolory i inne bajery mamy dodać dopiero w CSS-ach po kolejnych zajęciach.

Pozdrawiam, Agata

**Comment by Aimekram** [comment]
Z tego, co mówili na wykładzie to ma być tylko szielet HTML. Ale pytanie czy już klasy tworzyć i nadawać czy o tym też będzie później? Bo ktoś tam wcześniej o BEM pytał.

**Comment by Citrix94** [comment]
Wydaje mi się, że szkielet włącznie z klasami, obrazkami itp. możecie śmiało przygotować. Z tego co widać w planie zajęć na następnych zajęciach lecimy już z css, później flex i rwd, także grafik napięty :D

**Comment by nekorushi** [comment]
Robimy jedynie szkielet strony, więc tak - brzydka i biała :) Poprawne opisanie ich klasami jest również na plus, ale nie jest wymagane

**Comment by Aimekram** [comment]
Czy przy <img src="./images/band-member01.png"> ta kropka przed "images" odgrywa jakąś specjalną rolę? Na MDNie jest ścieżka bez kropki w przykładzie podana.

**Comment by AndriiChemerys** [comment]
Wydaje mi się, że "." (pojedyncza kropka) we względnym adresie URL oznacza pozostać w danym katalogu, natomiast ".." (dwie kropki) - wrócić do katalogu wyżej (rodzica).

Źródła: 
http://www.ee.surrey.ac.uk/Teaching/Unix/unix1.html
http://webreference.com/html/tutorial2/3.html

![dot](https://user-images.githubusercontent.com/38627095/48678962-84279180-eb8a-11e8-8757-ad95f9095fca.JPG)
